api/src/job.js

```diff
@@ -97,14 +97,18 @@ class Job {
     }
 
     async safe_call(file, args, timeout, memory_limit, eventBus = null) {
+        let { max_process_count, max_open_files, max_file_size, output_max_size } =
+            this.runtime.limit_overrides
+                ? this.runtime.limit_overrides
+                : config;
         return new Promise((resolve, reject) => {
             const nonetwork = config.disable_networking ? ['nosocket'] : [];
 
             const prlimit = [
                 'prlimit',
-                '--nproc=' + config.max_process_count,
-                '--nofile=' + config.max_open_files,
-                '--fsize=' + config.max_file_size,
+                '--nproc=' + (max_process_count || config.max_process_count),
+                '--nofile=' + (max_open_files || config.max_open_files),
+                '--fsize=' + (max_file_size || config.max_file_size),
```

Comment:
Shouldn't need to default to `config.*` here, ensure these variables are always populated